.travis.yml

```diff
@@ -3,21 +3,19 @@ services:
 - docker
 env:
   global:
-  - DIST_REPO="f5-sphinx-theme"
-  - USER_REPO=$(echo $TRAVIS_REPO_SLUG | cut -f -1 -d / )
-  - S3_DIST_URL="http://d3s2z99mfhr69y.cloudfront.net"
-  - BUILD_HTML_DIR=${TRAVIS_BUILD_DIR}/test-deploy/docs/_build/html
-  - BRANCH_DIR=${DIST_REPO}/travis-builds/${USER_REPO}/${TRAVIS_BRANCH}/
   - AWS_S3_BUCKET=${ARTIFACTS_BUCKET}
```

Comment:
`AWS_S3_BUCKET=${ARTIFACTS_BUCKET}` is no longer needed. I've set the bucket location in the repo settings.